docs/Server.md

```diff
@@ -0,0 +1,17 @@
+# Running the server
+Run as a thread (non blocking)

```

Comment:
Can't predict accurate use cases, since the server's execution should generally encompass the entire runtime. That is to say, there should never be a situation where its not running. The non-blocking syntax was added purely to demonstrate it's implementation of the Java `Runnable` interface